matchms/filtering/add_parent_mass.py

```diff
@@ -1,31 +1,58 @@
 from ..constants import PROTON_MASS
+from ..importing import load_adducts_table
 from ..typing import SpectrumType
 
 
-def add_parent_mass(spectrum_in: SpectrumType) -> SpectrumType:
-    """Add parentmass to metadata (if not present yet).
+def add_parent_mass(spectrum_in: SpectrumType, estimate_from_adduct: bool = True) -> SpectrumType:
+    """Add estimated parent mass to metadata (if not present yet).
 
-    Method to calculate the parent mass from given precursor m/z
-    and charge. Will take precursor m/z from either "precursor_mz"
+    Method to calculate the parent mass from given precursor m/z together
+    with charge and/or adduct. Will take precursor m/z from either "precursor_mz"
     or "pepmass" field.
+    For estimate_from_adduct=True this function will estimate the parent mass based on
+    the mass and charge of known adducts. The table of known adduct properties can be
+    found under matchms/data/known_adducts_table.csv.
+
+    Parameters
+    ----------
+    spectrum_in
+        Input spectrum.
+    estimate_from_adduct
+        When set to True, use adduct to estimate actual molecular mass ("parent mass").
+        Default is True. Still switches back to charge-based estimate if adduct does not match
```

Comment:
Still is not needed.
```suggestion
        Default is True. Switches back to charge-based estimate if adduct does not match
```